Documentation/Configuration/Index.rst

```diff
@@ -8,8 +8,9 @@
 
 .. _configuration:
 
+=============
```

Comment:
Are you really adding a line of "=" signs above the title? Is that a new practice for documentation? I'm used to adding such lines only after titles.

Reply:
Yep, for the title of a document (each rst file should have a document title):

See https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/HeadlinesAndSection.html

Reply:
It is a convention of the TYPO3 Documentation, not enforced by sphinx though